### ISSUE TITLE
Accordion & FAQ

### ISSUE THREAD

**Issue by sarah0arnold** [aemsites/clarkcountynv#1355]
**Component Block(s):** accordion + faq-accordion

**Design:** https://www.figma.com/design/T2bgro0OiuLqxvm1uLhJZ2/Clark-County-NV?node-id=132-735&t=7ZXrRUQI9M74s5dW-0

As a site visitor, I want the ability to drill down into a topic to learn more.

**Requirements:** 
Update accordion lines, spacing, title styling, and icon to match new design
When an accordion drawer is open, then caret icon should point down and both caret and title text should display in the primary color
When an accordion drawer is closed, then the caret should display pointing upwards and the caret and title text display in the #545454 color
There should be a horizontal line display before and after each accordion section (sandwich style: a line to start the component and end the component)
Accordions must be keyboard navigable - user can open/close drawers of the accordion and access content within the accordion drawer when necessary via keyboard navigation
Accordions must be screen readable: content within the accordion should be readable by a screen reader and follow logical block ordering when announced
Leverage styling for the documents center search bar for the search field: 
- Placeholder 'Search' box text should be #828282 color
- User should be able to input a search criteria, clear a search input, and navigate faq filtered results/accordion drawer content via keyboard navigation

**Acceptance Criteria:** 
1. Given I am viewing a page with an accordion, when the page renders, then the elements within the accordion are styled to match the design. 
2. Given I am viewing a page with an accordion, when I click to open an accordion drawer, then the caret icon change direction and the caret and title text are both primary color. 
3. Given I am viewing a page with an accordion, when some of the accordion drawers are closed, then the caret icon displays pointing upwards (collapsed state) and the caret and title text are #545454 color.
4. Given I am viewing a page with an accordion, when the component block renders, then there are horizontal line breaks before, between, and after each accordion drawer (single line in each position)
5. Given I am viewing a page with an accordion, when I use my keyboard to navigate the page, then I am able to interact with the accordion drawers (expand/collapse) and access content within the accordion drawer. 
6. Given I am using a screen reader, when I navigate a page with an accordion, then the accordion content is accessible and read in a logical order.

DECISION: Mimic search design from documents center for styling requirements

**Comment by beccasuchanek** [comment]
@aaronstewart-dev This ticket needs review for collapsed accordion styling acceptance criteria, see details below:

Test Page URL: https://develop--clarkcountynv--aemsites.aem.page/drafts/bsuchanek/tickets/accordion-and-faq
Figma Design Link: https://www.figma.com/design/T2bgro0OiuLqxvm1uLhJZ2/Clark-County-NV?node-id=132-735&t=LKjWnm3ltylPxKxC-4

3. ❌ Given I am viewing a page with an accordion, when some of the accordion drawers are closed, then the caret icon displays pointing upwards (collapsed state) and the caret and title text are #545454 color. **Happening on accordion block but not nested accordion**

<img width="1383" height="582" alt="Image" src="https://github.com/user-attachments/assets/48931ec3-41d3-4e96-b5e8-d0f0346163b2" />

**Status of ACs**
1. ✅ Given I am viewing a page with an accordion, when the page renders, then the elements within the accordion are styled to match the design. 

<img width="757" height="821" alt="Image" src="https://github.com/user-attachments/assets/ccfd6d60-54aa-46d2-83ba-bca8362e7c6c" />

2. ✅ Given I am viewing a page with an accordion, when I click to open an accordion drawer, then the caret icon change direction and the caret and title text are both primary color. 

<img width="1008" height="774" alt="Image" src="https://github.com/user-attachments/assets/30aec1ec-5ff9-4d52-b6ad-f9fe974a604d" />

3. ❌ Given I am viewing a page with an accordion, when some of the accordion drawers are closed, then the caret icon displays pointing upwards (collapsed state) and the caret and title text are #545454 color.
4. ✅ Given I am viewing a page with an accordion, when the component block renders, then there are horizontal line breaks before, between, and after each accordion drawer (single line in each position)

<img width="773" height="656" alt="Image" src="https://github.com/user-attachments/assets/73b03de3-51a0-4231-b41e-dbce3a90a2d3" />

<img width="961" height="544" alt="Image" src="https://github.com/user-attachments/assets/4e0bd606-cb69-40fc-859a-5de2151a46e3" />

5. ✅ Given I am viewing a page with an accordion, when I use my keyboard to navigate the page, then I am able to interact with the accordion drawers (expand/collapse) and access content within the accordion drawer. Passed but video too large, if you want to see it visit this link to download/watch https://twodegrees1.sharepoint.com/:v:/r/teams/ClarkCounty131/Shared%20Documents/General/EDS%20QA/Proof%20of%20Testing%20Videos/Two%20Column%20Text%20Layout_Screenreader.mov?csf=1&web=1&e=VZgIYR
6. ✅ Given I am using a screen reader, when I navigate a page with an accordion, then the accordion content is accessible and read in a logical order. Passed but video too large, if you want to see it visit this link to download/watch https://twodegrees1.sharepoint.com/:v:/r/teams/ClarkCounty131/Shared%20Documents/General/EDS%20QA/Proof%20of%20Testing%20Videos/Two%20Column%20Text%20Layout_Screenreader.mov?csf=1&web=1&e=VZgIYR

**Comment by aaronstewart-dev** [comment]
@beccasuchanek Latest on develop for accordion styling issues. Assigning QA label back.

**Comment by beccasuchanek** [comment]
@aaronstewart-dev This ticket passed all ACs, see details below:

**Test Page URL:** https://develop--clarkcountynv--aemsites.aem.page/drafts/bsuchanek/tickets/accordion-and-faq

**Status of ACs**
1. ✅ Given I am viewing a page with an accordion, when the page renders, then the elements within the accordion are styled to match the design. **See in previous comment**
2. ✅ Given I am viewing a page with an accordion, when I click to open an accordion drawer, then the caret icon change direction and the caret and title text are both primary color. **See in previous comment**
3. ✅ Given I am viewing a page with an accordion, when some of the accordion drawers are closed, then the caret icon displays pointing upwards (collapsed state) and the caret and title text are #545454 color. 

<img width="1365" height="370" alt="Image" src="https://github.com/user-attachments/assets/c67f1aca-546f-49c9-83e5-06ac6d3e7c8d" />

<img width="1379" height="578" alt="Image" src="https://github.com/user-attachments/assets/ff98558e-3f06-4b9a-ad88-11f2aed8118e" />

4. ✅ Given I am viewing a page with an accordion, when the component block renders, then there are horizontal line breaks before, between, and after each accordion drawer (single line in each position) **See in previous comment**
5. ✅ Given I am viewing a page with an accordion, when I use my keyboard to navigate the page, then I am able to interact with the accordion drawers (expand/collapse) and access content within the accordion drawer. **See in previous comment**
6. ✅ Given I am using a screen reader, when I navigate a page with an accordion, then the accordion content is accessible and read in a logical order. **See in previous comment**
